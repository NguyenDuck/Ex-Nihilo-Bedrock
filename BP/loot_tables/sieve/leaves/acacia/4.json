{
    "pools": [
        {
            "rolls": 1,
            "entries": [
                {
                    "type": "loot_table",
                    "name": "loot_tables/sieve/leaves/acacia/3.json"
                }
            ]
        },
        {
            "rolls": 1,
            "entries": [
                {
                    "type": "item",
                    "name": "sapling",
                    "functions": [
                        {
                            "function": "set_data",
                            "data": 4
                        }
                    ],
                    "weight": 20
                },
                {
                    "type": "empty",
                    "weight": 80
                }
            ]
        }
    ]
}