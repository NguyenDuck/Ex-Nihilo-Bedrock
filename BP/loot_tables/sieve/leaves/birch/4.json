{
    "pools": [
        {
            "rolls": 1,
            "entries": [
                {
                    "type": "loot_table",
                    "name": "loot_tables/sieve/leaves/birch/3.json"
                }
            ]
        },
        {
            "rolls": 1,
            "entries": [
                {
                    "type": "item",
                    "name": "sapling",
                    "functions": [
                        {
                            "function": "set_data",
                            "data": 2
                        }
                    ],
                    "weight": 20
                },
                {
                    "type": "empty",
                    "weight": 80
                }
            ]
        }
    ]
}