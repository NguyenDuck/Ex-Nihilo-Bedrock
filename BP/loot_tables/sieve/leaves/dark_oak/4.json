{
    "pools": [
        {
            "rolls": 1,
            "entries": [
                {
                    "type": "loot_table",
                    "name": "loot_tables/sieve/leaves/dark_oak/3.json"
                }
            ]
        },
        {
            "rolls": 1,
            "entries": [
                {
                    "type": "item",
                    "name": "sapling",
                    "functions": [
                        {
                            "function": "set_data",
                            "data": 5
                        }
                    ],
                    "weight": 20
                },
                {
                    "type": "empty",
                    "weight": 80
                }
            ]
        }
    ]
}