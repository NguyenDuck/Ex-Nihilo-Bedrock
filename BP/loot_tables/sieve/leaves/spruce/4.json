{
    "pools": [
        {
            "rolls": 1,
            "entries": [
                {
                    "type": "loot_table",
                    "name": "loot_tables/sieve/leaves/spruce/3.json"
                }
            ]
        },
        {
            "rolls": 1,
            "entries": [
                {
                    "type": "item",
                    "name": "sapling",
                    "functions": [
                        {
                            "function": "set_data",
                            "data": 1
                        }
                    ],
                    "weight": 20
                },
                {
                    "type": "empty",
                    "weight": 80
                }
            ]
        }
    ]
}