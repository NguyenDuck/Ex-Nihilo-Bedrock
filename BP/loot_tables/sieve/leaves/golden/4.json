{
    "pools": [
        {
            "rolls": 1,
            "entries": [
                {
                    "type": "loot_table",
                    "name": "loot_tables/sieve/leaves/golden/3.json"
                }
            ]
        },
        {
            "rolls": 1,
            "entries": [
                {
                    "type": "item",
                    "name": "apple",
                    "weight": 20
                },
                {
                    "type": "empty",
                    "weight": 80
                }
            ]
        },
        {
            "rolls": 1,
            "entries": [
                {
                    "type": "item",
                    "name": "golden_apple",
                    "weight": 1
                },
                {
                    "type": "empty",
                    "weight": 99
                }
            ]
        },
        {
            "rolls": 1,
            "entries": [
                {
                    "type": "item",
                    "name": "exnihilo:silkworm",
                    "weight": 20
                },
                {
                    "type": "empty",
                    "weight": 80
                }
            ]
        }
    ]
}